### PR TITLE
Course lessons

### PR DESCRIPTION
**Pull request by Dimanaux**

### CODE REVIEW

File: app/controllers/api/v1/courses/lessons_controller.rb [hunk 1]
```diff
@@ -0,0 +1,58 @@
+module Api
+  module V1
+    module Courses
+      class LessonsController < ApplicationController
+        expose :course
+        expose :lesson, parent: :course, find_by: :order
+        expose :lessons, from: :course
+
+        before_action only: :show do
+          self.class.serialization_scope :detailed
```
Comment (RamilyaNigmatullina) [inline app/controllers/api/v1/courses/lessons_controller.rb]: Why do we need this code?

File: app/controllers/api/v1/courses/enrollments_controller.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
+module Api
+  module V1
+    module Courses
+      class EnrollmentsController < ApplicationController
+        expose :course
+        expose :enrollment, -> { Enrollment.find_by(enrollment_params) }
```
Comment (RamilyaNigmatullina) [inline app/controllers/api/v1/courses/enrollments_controller.rb]: Why can't we find enrollment by id?

File: db/migrate/20200728124743_create_users.rb [hunk 3]
```diff
@@ -4,8 +4,8 @@ def change
 
     create_table :users do |t|
       t.citext :email, null: false
-      t.string :full_name, null: false, default: ""
-      t.string :password_digest
+      t.string :full_name, null: false
+      t.string :password_digest, null: false
```
Comment (RamilyaNigmatullina) [inline db/migrate/20200728124743_create_users.rb]: Is it ok that we change existing migration?

**Review by RamilyaNigmatullina — APPROVED** [review]
I left some comments. But PR looks good for me.

File: app/controllers/api/v1/courses/enrollments_controller.rb (hunk 2 above)
Comment (DmitryBarskov) [reply app/controllers/api/v1/courses/enrollments_controller.rb]: We can't because there is no id passed as a path variable

**Pull request merged**